test/models/notice_test.rb

```diff
@@ -450,7 +450,7 @@
       end
 
       it "tag.nameが両方カンマ区切りで含まれていること" do
-        Notice.find(@notice).tags_string.must_equal @tags.map {|tag| tag.name }.join(",")
+        Notice.find(@notice.id).tags_string.must_equal @tags.map {|tag| tag.name }.join(",")
```

Comment:
Line is too long. [92/80]<br>Space between { and | missing.<br>Pass `&:name` as an argument to `map` instead of a block.